bubuku/broker.py

```diff
@@ -24,8 +25,42 @@ def is_running_and_registered(self):
         return self.id_manager.is_registered()
 
     def stop_kafka_process(self):
+        """
+        Stops kafka process (if it is running) and says if this topic is still is a leader on in ISR list for some
+        partitions
+        :return: True, if broker is stopped and is not a leader or a isr.
+        """
         self._terminate_process()
         self._wait_for_zk_absence()
+        return not self._have_leadership()
+
+    def _is_clean_election(self):
```

Comment:
Ok, "Let me speak from my heart". At any moment when someone creates property that contains "not" in it's name - someone have to suffer. Right now it's me.
For me (and for everyone, i think) it's simpler not to have negation in property name, so it's better not to have 'not not' if one checking positive condition